app/templates/components/forms/orders/order-form.hbs

```diff
@@ -223,16 +223,14 @@
                     <span>{{t 'I agree to the terms of the'}} </span>
                     <a href="https://foundation.wikimedia.org/wiki/Policy:Universal_Code_of_Conduct">{{t 'Universal Code of Conduct'}}</a>
                     <span> {{t 'and the'}} </span>
-                    <a href="https://foundation.wikimedia.org/wiki/Policy:Friendly_space_policy">{{t 'Friendly Space Policy'}}</a>
-                    <span> .</span>
+                    <a href="https://foundation.wikimedia.org/wiki/Policy:Friendly_space_policy">{{t 'Friendly Space Policy'}}.</a>
```

Comment:
previous was correct; only to remove space before the full stop so please revert to
<span>.</span>